browser/ui/views/tabs/brave_tab_strip.cc

```diff
@@ -388,7 +386,8 @@ void BraveTabStrip::UpdateTabStripMargins() {
 
   gfx::Insets margins;
 
-  if (!ShouldShowVerticalTabs()) {
+  if (!ShouldShowVerticalTabs() && !GetWidget()->IsMaximized() &&
```

Comment:
In the future, we might want to consider moving this padding out into the platform-specific browser frame view layout code. However, I thought it would be best to fix the issue here for now.

Reply:
Nice. I think it'd be better to check GetWidget() validity just in case.

Reply:
Margin setting logic has been moved to `brave_browser_view_layout.cc`.